app/assets/javascripts/admin/modules/document_finder.js

```diff
@@ -13,89 +13,89 @@
     // 'document_id') or edition's id (with a name attribute of 'edition_id').
 
     init: function (params) {
-      this.no_results_message = params.no_results_message || 'No results matching search criteria'
+      this.noResultsMessage = params.no_results_message || 'No results matching search criteria'
```

Comment:
You could update this to `noResultsMessage` if you update [this line](https://github.com/alphagov/whitehall/blob/master/app/views/admin/statistics_announcements/_document_linker.html.erb#L2), although I appreciate it's safer not to. I'd perhaps put something in the commit to say whether you've considered it.

Reply:
Thanks - I did decide against editing that as it felt like we expected this to be called from Ruby where it's more idiomatic to have snake_case. It is a pain when you're at the point where the interface conventions interact.